test-cases/longevity/longevity-twcs-48h-tombstone-gc.yaml

```diff
@@ -0,0 +1,34 @@
+test_duration: 3000
```

Comment:
do you have a staging run of this job?

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/4755#issuecomment-1144734793

Reply:
i see both jobs failed...